static/sass/_snapcraft_tour.scss

```diff
@@ -0,0 +1,243 @@
+@mixin snapcraft-tour {
+  $color-overlay: rgba(0, 0, 0, .5);
+  $triangle-height: $sp-unit;
+
+  // some temporary testing overlay
+  body {
+    position: relative;
+  }
+
+  .p-tour-overlay {
+    background: $color-overlay;
+    bottom: 0;
+    left: 0;
+    position: absolute;
+    right: 0;
+    top: 0;
+    z-index: 9999;
+  }
+
+  // triange arrow for tooltips
+  %triangle {
+    &::before,
+    &::after {
+      border: {
+        bottom: $triangle-height solid transparent;
+        left: $triangle-height solid transparent;
+        right: $triangle-height solid transparent;
+        top: $triangle-height solid transparent;
+      }
+      content: '';
+      height: 0;
+      pointer-events: none;
+      position: absolute;
+      width: 0;
+    }
+  }
+
+  // for tooltips below element
+  %triangle--pointing-up {
+    @extend %triangle;
+
+    &::before,
+    &::after {
+      border-bottom-color: $color-mid-light;
+      bottom: 100%;
+    }
+
+    &::after {
+      border-bottom-color: $color-x-light;
+      bottom: calc(100% - 1px);
+    }
+  }
+
+  // for tooltips above element
+  %triangle--pointing-down {
+    @extend %triangle;
+
+    &::before,
+    &::after {
+      border-top-color: $color-mid-light;
+      top: 100%;
+    }
+
+    &::after {
+      border-top-color: $color-x-light;
+      top: calc(100% - 1px);
+    }
+  }
+
+  // horizontal placement of up/down arrow
+  %triangle--on-left {
+    &::before,
+    &::after {
+      left: $sph-inner;
+    }
+  }
+
+  %triangle--on-right {
+    &::before,
+    &::after {
+      right: $sph-inner;
+    }
+  }
+
+  %triangle--on-center { // horizontally centered, as in text-align
```

Comment:
OK, final suggestion - what if we referred to the horizontal and vertical axes X and Y, i.e.
`%triangle--x-center`
`%triangle--y-center`